Dockerfile

```diff
@@ -0,0 +1,107 @@
+FROM ubuntu:20.04
+
+LABEL "name"="Statick"
+LABEL "version"="0.0"
+LABEL "repository"="https://github.com/sscpac/statick-action.git"
+LABEL "homepage"="https://github.com/sscpac/statick-action"
+LABEL "maintainer"="Thomas Denewiler <tdenewiler@gmail.com>"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+USER root
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    cccc \
+    chktex \
+    clang \
+    clang-6.0 \
+    clang-10 \
+    clang-format \
+    clang-format-6.0 \
+    clang-format-10 \
+    clang-tidy \
+    clang-tidy-6.0 \
+    clang-tidy-10 \
+    cmake \
+    cppcheck \
+    curl \
+    file \
+    findbugs \
+    flawfinder \
+    git \
+    lacheck \
+    libomp-dev \
+    libpcre3-dev \
+    libperl-critic-perl \
+    libxml2 \
+    libxml2-utils \
+    maven \
+    pylint \
```

Comment:
We might be able to remove this from `apt-get install` since we get an updated version from PyPI with `pip install`. That could be a separate issue later.

Same with `flawfinder`.

Reply:
Will handle in the PR that moves the Dockerfile out of this repo.